### PR TITLE
feat(frontend): Show Pipeline Spec in Pipeline Detail page. Fix #6279

### LINKED ISSUE #6279: Show json payload of the Pipeline definition template (instead of YAML).

### PR DESCRIPTION
**Pull request by zijianjoy**
**Description of your changes:**

Fix #6279

pipelinespectab

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: frontend/src/pages/PipelineDetailsV2.tsx [hunk 1]
```diff
@@ -14,37 +14,66 @@
...
-      <div className={commonCss.page} style={{ position: 'relative', overflow: 'hidden' }}>
-        <StaticCanvas
-          layers={layers}
-          onLayersUpdate={layerChange}
-          elements={pipelineFlowElements}
-        ></StaticCanvas>
-      </div>
+      <MD2Tabs
+        selectedTab={selectedTab}
+        onSwitch={(tab: number) => setSelectedTab(tab)}
```
Comment (Bobgy) [inline frontend/src/pages/PipelineDetailsV2.tsx:48]: nit: you can pass the original function directly here.

onSwitch={setSelectedTab}

File: frontend/src/pages/PipelineDetailsV2.tsx [hunk 2]
```diff
@@ -14,37 +14,66 @@
...
 
-const PipelineDetailsV2: React.FC<PipelineDetailsV2Props> = ({
+function PipelineDetailsV2({
+  templateString,
   pipelineFlowElements,
   setSubDagLayers,
-}: PipelineDetailsV2Props) => {
+}: PipelineDetailsV2Props) {
   const [layers, setLayers] = useState(['root']);
+  const [selectedTab, setSelectedTab] = useState(0);
```
Comment (Bobgy) [inline frontend/src/pages/PipelineDetailsV2.tsx:36]: nit: write constants for tabs

**Review by Bobgy — COMMENTED** [review]
/lgtm
Only a few nit pickings.
This looks great!

File: frontend/src/pages/PipelineDetailsV2.tsx (hunk 1 above)
Comment (zijianjoy) [reply frontend/src/pages/PipelineDetailsV2.tsx:48]: Sounds good! Done.

File: frontend/src/pages/PipelineDetailsV2.tsx (hunk 2 above)
Comment (zijianjoy) [reply frontend/src/pages/PipelineDetailsV2.tsx:36]: Added constants for tabs.

**Pull request merged**